settings-parent.gradle

```diff
@@ -0,0 +1,6 @@
+apply from: 'settings.gradle'
+
+rootProject.children.each { project ->
+    String relativeProjectPath = project.projectDir.path.replace(settingsDir.path, "")
+    project.projectDir = new File("vimeo-networking-java/$relativeProjectPath")
+}
```

Comment:
This basically fixes the relative paths of nested submodules, so that we can have nested submodules without needing to modify references in the build.gradle files from `compile project(':subproject')` to `compile project('parent:subproject')`